core/modules/views/views.install

```diff
@@ -186,6 +186,21 @@ function views_update_1006() {
   }
 }
 
+/**
+ * Remove the unused "core" key from active config files.
+ */
+function views_update_1007() {
+  $configs = config_get_names_with_prefix('views.view.');
+
+  foreach ($configs as $config_file) {
+    $core_key_in_config = config_get($config_file, 'core');
```

Comment:
Would it be better to use:

```php
$config = config($config_file);
$core_key_in_config = $config->get('core');
```

here to avoid reading the same config file twice (once for getting the value and once for clearing it)?

Reply:
You're right. Done...